src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dayRent = 40;
+  let totalRental = 0;
```

Comment:
Prefer `const` over `let` where possible. Since `totalRental` is only assigned once in each condition, you can use `const` within each block instead of `let` at the beginning, as per checklist item 3.